Dockerfile

```diff
@@ -9,29 +9,27 @@ RUN cd /tmp && \
 
 ################################################################################
 
-FROM registry.access.redhat.com/ubi8/ubi-minimal:latest AS postgresql_container_source
+FROM registry.access.redhat.com/ubi9/ubi-minimal:latest AS postgresql_container_source
 
 RUN microdnf -y --setopt=tsflags=nodocs install git
 RUN git clone --branch generated --depth 1 https://github.com/sclorg/postgresql-container /postgresql-container
 
 ################################################################################
 
-FROM registry.access.redhat.com/ubi8/s2i-core AS base
+FROM registry.access.redhat.com/ubi9/s2i-core as base
 
 # PostgreSQL image for OpenShift.
 # Volumes:
-#  * /var/lib/psql/data   - Database cluster for PostgreSQL
+#  * /var/lib/pgsql/data  - Database cluster for PostgreSQL
```

Comment:
Is this just a typo or did the actual dir move?

Reply:
It was just a typo